interface/usergroup/user_info_ajax.php

```diff
@@ -45,8 +36,8 @@
 $errMsg='';
 $success=update_password($_SESSION['authId'], $_SESSION['authId'], $curPass, $newPass, $errMsg);
 if ($success) {
-    echo xlt("Password change successful");
+    echo "<div class='alert alert-success'>" . xlt("Password change successful") . "</div>";
```

Comment:
@robertdown ,
Ok, I think I sorted this out, by migrating the alert css to alert-custom (it is only used in a couple places) which then opened the door to being able to use the bootrstrap alert classes. just pushed the commit on this pr. recommend viewing without whitespace changes via the ?w=1 parameter:
https://github.com/openemr/openemr/pull/950/files?w=1